bundles/org.openhab.binding.pjlinkdevice/src/main/java/org/openhab/binding/pjlinkdevice/internal/device/command/lampstatus/LampStatesResponse.java

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.pjlinkdevice.internal.device.command.lampstatus;
+
+import org.openhab.binding.pjlinkdevice.internal.device.command.PrefixedResponse;
+import org.openhab.binding.pjlinkdevice.internal.device.command.ResponseException;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.NoSuchElementException;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The response part of {@link LampStatesCommand}
+ *
+ * @author Nils Schnabel - Initial contribution
+ */
+@NonNullByDefault
+public class LampStatesResponse extends PrefixedResponse<List<LampStatesResponse.LampState>> {
+  @NonNullByDefault
+  public class LampState {
+    private boolean active;
+    private int lampHours;
+
+    public LampState(boolean active, int lampHours) {
+      this.active = active;
+      this.lampHours = lampHours;
+    }
+
+    public int getLampHours() {
+      return lampHours;
+    }
+
+    public boolean isActive() {
+      return active;
+    }
+  }
+
+  public LampStatesResponse(String response) throws ResponseException {
+    super("LAMP=", response);
+  }
+
+  @Override
+  protected List<LampStatesResponse.LampState> parseResponseWithoutPrefix(String responseWithoutPrefix)
```

Comment:
I did that before, but this blows up the diff as almost all files of the binding will be touched. Best practice is IMHO to separate content and formatting changes into different commits/PRs, so that you can have an undisturbed look at content changes. I could do another PR to get the whole binding to the new formatting afterwards, what do you think?

Reply:
Sounds like a good idea!